lib/scrub/autoscrub.rb

```diff
@@ -22,6 +22,15 @@ class AutoScrub
     def initialize(path)
       @path = path
 
+      # @member_id and @item_type are used in the path to scrub_logger, but we
+      # also need somewhere to log to before we know @member_id and @item_type
```

Comment:
What is the purpose of the early logging? If it's primarily for our own debugging, could it only go to STDOUT and we look at it via loki, or are there reasons to want it in a file? (I know there are many unknowns with loki, plus it only currently retains logs for I think 14 days, but I'd still like to understand what we end up doing with this 'early' log file)